### PR TITLE
Finalize release notes for v1.6.0 final

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @TomSweeneyRedHat @giuseppe @baude PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -43,13 +43,20 @@
 - Fixed a bug where removing an unmounted container that was unmounted might sometimes not properly clean up the container ([#4033](https://github.com/containers/libpod/issues/4033))
 - Fixed a bug where the Varlink server would freeze when run in a systemd unit file ([#4005](https://github.com/containers/libpod/issues/4005))
 - Fixed a bug where Podman would not properly set the `$HOME` environment variable when the OCI runtime did not set it
+- Fixed a bug where rootless Podman would incorrectly print warning messages when an OCI runtime was not found ([#4012](https://github.com/containers/libpod/issues/4012))
+- Fixed a bug where named volumes would conflict with, instead of overriding, `tmpfs` filesystems added by the `--read-only-tmpfs` flag to `podman create` and `podman run`
+- Fixed a bug where `podman cp` would incorrect make the target directory when copying to a symlink which pointed to a nonexistent directory ([#3894](https://github.com/containers/libpod/issues/3894))
```
Comment (baude) [inline RELEASE_NOTES.md]: incorrectly

**Pull request merged**